### PR TITLE
Updated content under public org heading on membership join us page

### PR DESCRIPTION
**Pull request by ElenaFdR**
This links to the new 'join us as a founding member' page on the marketing site. 
Here's the draft PR for that page:
https://github.com/publiccodenet/http/pull/22

This page partially fixes issue 20 on the http repo: https://github.com/publiccodenet/http/issues/20.

I'd love feedback on this, especially the final sentence (which seems a bit contrived to me).

Feel free to untag yourself as a reviewer.
(The commit history looks so intense because this branch is based on the new About hierarchy - the final commit on this page is the only one that I'm requesting feedback on).

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -25,9 +25,9 @@ We’re especially interested in learning what you need to implement the [Standa
 
 ## Public organizations
 
-We’re currently looking for innovative public organizations to join us as founding members. As a founding member, you’ll have the chance to shape the Foundation’s processes and governance. Your code will be among the first accepted for Foundation stewardship. Lead the community of enterprising public organizations creating public code to power their digital transformations.
+We’re currently looking for innovative public organizations to [join us as founding members](https://www.publiccode.net/founding-membership). As a founding member, you’ll have the chance to shape the Foundation’s processes and governance. Your code will be among the first accepted for Foundation stewardship.
```
Comment (bvhme) [inline CONTRIBUTING.md]: Make sure this link exists before pulling this pull request. Travis now throws an error because it is broken.

Comment (ElenaFdR) [reply CONTRIBUTING.md]: http://publiccode.net/membership/founding-membership is now live

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -15,15 +19,15 @@ In the meantime, please join us in any way you can.
...
 
 We’re especially interested in learning what you need to implement the [Standard for Public Code](http://standard.publiccode.net/) in your organization. Please [raise an issue](http://standard.publiccode.net/CONTRIBUTING.html) or email us at <info@publiccode.net>.
 
 ## Public organizations
 
-We’re currently looking for innovative public organizations to join us as founding members. As a founding member, you’ll have the chance to shape the Foundation’s processes and governance. Your code will be among the first accepted for Foundation stewardship. Lead the community of enterprising public organizations creating public code to power their digital transformations.
+We’re currently looking for innovative public organizations to [join us as founding members](https://www.publiccode.net/founding-membership). As a founding member, you’ll have the chance to shape the Foundation’s processes and governance. Your code will be among the first accepted for Foundation stewardship.
 
-If this sounds like your organization, email us at <membership@publiccode.net>.
+Come lead our community of enterprising public organizations creating public code to power their digital transformations.
```
Comment (ElenaFdR) [inline CONTRIBUTING.md]:
```suggestion
Come lead our community of enterprising public organizations using public code to power their digital transformations.
```

Comment (bvhme) [reply CONTRIBUTING.md]: Great, love it.

**Comment by bvhme** [comment]
Please update this branch from Develop, it will make your life easier and easier to see the changes you've made. There is a button above where the merge button usually is.

File: CONTRIBUTING.md (hunk 1 above)
Comment (ElenaFdR) [inline CONTRIBUTING.md]:
```suggest
We’re currently looking for innovative public organizations to [join us as founding members](https://www.publiccode.net/membership/founding-membership). As a founding member, you’ll have the chance to shape the Foundation’s processes and governance. Your code will be among the first accepted for Foundation stewardship.
```

**Pull request merged**